### PR TITLE
feat: installed and set up preline UI

### PR DESCRIPTION
**Pull request by dev-phantom**
## Description
installed and set up preline UI

## Related Issue

Resolves: #133

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -16,6 +16,10 @@ function MyApp({ Component, pageProps }: ComponentWithPageLayout) {
   // From documentation on using Posthog with Next.js: https://posthog.com/docs/integrate/third-party/next-js
   const router = useRouter();
 
+  useEffect(() => {
+    import('preline')
```
Comment (chadstewart) [inline src/pages/_app.tsx:20]: So there seems to be an issue with the deployment. Seems like TypeScript can't find type definitions for this import for some reason. Could you look into that and see if you can resolve it please?

**Review by chadstewart — CHANGES_REQUESTED** [review]
Hey @dev-phantom, thanks much for the contribution!

Just need to clear up an issue that I mention in the review. Thanks much!

File: src/pages/_app.tsx (hunk 1 above)
Comment (dev-phantom) [reply src/pages/_app.tsx:20]: > So there seems to be an issue with the deployment. Seems like TypeScript can't find type definitions for this import for some reason. Could you look into that and see if you can resolve it please?

this issue has been resolved in my last commit

**Comment by chadstewart** [comment]
I'm really sorry about this but decided that I'm going to go with ShadCN instead of preline so this PR is no longer needed.

Really sorry but thank you so much for your contribution.

**Pull request closed without merging**